shell/edit/cloudcredential.vue

```diff
@@ -272,47 +274,46 @@ export default {
 </script>
 
 <template>
-  <form class="filled-height">
```

Comment:
Removing this to be consistent with all other uses of CruResource. I can put this back in if we'd like. It doesn't affect the footer which was the original purpose.

Reply:
I'm fine with keeping this out, the extra form tag appeared to be a little redundant in the dom tree.